docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -349,6 +349,27 @@
             deploy_config: "/home/{{ ansible_ssh_user }}/deployment-config.yaml"
           when: inventory_hostname != 'localhost'
 
+         # Configure required region value for subcloud
+        - name: Get subcloud region value from environment
+          shell: source /etc/platform/openrc; echo $OS_REGION_NAME
+          register: os_region_name_from_env
+
+        - name: Get subcloud region value from deployment file
+          shell: grep OS_REGION_NAME {{deploy_config}}
+          register: os_region_name_from_deploy_file
+
+        - name: Show subcloud region
+          debug:
+            msg:
+            - "Subcloud region value : {{os_region_name_from_env.stdout}}"
+
+        - name: Configure new subcloud region value
+          replace:
+            path: "{{deploy_config}}"
+            regexp: '{{os_region_name_from_deploy_file.stdout | trim}}'
```

Comment:
The OS_REGION_NAME is kind of static, no need to have the task in line357 and print it out here